spec/integration/until_and_while_executing_spec.rb

```diff
@@ -3,12 +3,14 @@
 RSpec.describe "SidekiqUniqueJobs::Lock::UntilAndWhileExecuting" do
   before do
     digests.delete_by_pattern("*")
+    toxic_redis_url = ENV["CI"] ? "toxiproxy:21212" : "localhost:21212"
```

Comment:
I suspect you'll also need the following for CI to properly point to the toxiproxy daemon container/service:
```ruby
Toxiproxy.host = 'http://toxiproxy:8474' if ENV["CI"]
```

I'd imagine this'll resolve the current `Failed to open TCP connection to 127.0.0.1:8474` errors in CI.

Reply:
> Failed to open TCP connection to toxiproxy:8474 (getaddrinfo: Temporary failure in name resolution)

🤷‍♂️ 

Well I got it working locally so I'll skip this spec for CI for now.